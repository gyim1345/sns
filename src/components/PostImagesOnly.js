import React from 'react';

function PostImagesOnly({ imageUrl }) {
  return (
    <div
      className="test"
      style={{
        position: 'relative',
        width: '17vw',
        marginRight: '2%',
        marginBottom: '2%'
      }}
    >
      <div
        style={{
          position: 'absolute',
          top: 0,
          left: 0,
          bottom: 0,
          right: 0,
          backgroundImage: `url(${imageUrl})`,
          backgroundSize: 'cover',
          backgroundPosition: 'center',
          marginRight: '2%'
        }}
      ></div>
      <div style={{ paddingTop: '100%' }}></div>
    </div>
  );
}

export default PostImagesOnly;
