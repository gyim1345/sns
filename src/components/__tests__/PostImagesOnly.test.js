import React from 'react';
import { mount } from 'enzyme';
import PostImagesOnly from '../PostImagesOnly';
import { info } from './testjs';

describe('<PostImagesOnly />', () => {
  it('스냅샷 비교', () => {
    const wrapper = mount(<PostImagesOnly imageUrl={'URL'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
