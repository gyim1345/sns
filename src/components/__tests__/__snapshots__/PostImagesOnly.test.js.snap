// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostImagesOnly /> 스냅샷 비교 1`] = `
<PostImagesOnly
  imageUrl="URL"
>
  <div
    className="test"
    style={
      Object {
        "marginBottom": "2%",
        "marginRight": "2%",
        "position": "relative",
        "width": "26vw",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundImage": "url(URL)",
          "backgroundPosition": "center",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "marginRight": "2%",
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
    <div
      style={
        Object {
          "paddingTop": "100%",
        }
      }
    />
  </div>
</PostImagesOnly>
`;
